.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: false,
```

Comment:
Who needs semicolons anyway ;)

The very few edge cases where JavaScripts ASI fails are so rare that they're not even worth discussing in my opinion.

Reply:
Yeah, semi-colons make code unaesthetic in my eyes anyway. 🖌